modules/bbp-morphology/src/test/resources/data/bbp/morphology/fixstainmount/v0.1.0/min-fields.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
```

Comment:
The context "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0" is for schema.